internal/config/config.go

```diff
@@ -542,6 +542,11 @@ func (c *Config) IsReleaseImage() bool {
 	return strings.HasPrefix(c.Image, "v")
 }
 
+// IsDebugImage checks whether image name looks like a debug image.
+func (c *Config) IsDebugImage() bool {
+	return strings.Contains(c.Image, "debug")
+}
```

Comment:
we may want to parse the image using `github.com/edgelesssys/constellation/v2/internal/api/versionsapi.NewVersionFromShortPath` and compare the `version.Stream` to `debug`.